pika/adapters/twisted_connection.py

```diff
@@ -405,6 +408,32 @@ def _send_frame(self, frame_value):
         self.bytes_sent += len(marshaled_frame)
         self.frames_sent += 1
         self.transport.write(marshaled_frame)
+        
+# fixed here
+    def _send_message(self, channel_number, method_frame, content=None):
```

Comment:
This looks pretty similar to `Connection._send_message`. Is there any reason why a single implementation cannot be shared?

CC @wjps